applications/ShapeOptimizationApplication/tests/total_volume/test_total_volume_response.py

```diff
@@ -0,0 +1,53 @@
+import KratosMultiphysics as Kratos
+import KratosMultiphysics.KratosUnittest as UnitTest
+from KratosMultiphysics.ShapeOptimizationApplication.response_functions.total_volume import TotalVolume
+
+
+class TestTotalVolumeResponseFunction(UnitTest.TestCase):
+    @classmethod
+    def setUpClass(cls):
+        cls.model = Kratos.Model()
+        cls.model_part = cls.model.CreateModelPart("test")
+        cls.model_part.CreateNewNode(1, 0.0, 0.0, 0.0)
+        cls.model_part.CreateNewNode(2, 1.0, 0.0, 0.0)
+        cls.model_part.CreateNewNode(3, 1.0, 1.0, 0.0)
+
+        prop = cls.model_part.GetProperties()[0]
+        cls.model_part.CreateNewElement("Element2D3N", 1, [2, 3, 1], prop)
+
+        cls.model_part.ProcessInfo[Kratos.DOMAIN_SIZE] = 2
+
+        cls.response_function = TotalVolume(1, Kratos.Parameters("""{
+            "response_type"            : "total_volume",
+            "model_part_name"          : "test",
+            "output_variable_name"     : "SHAPE_SENSITIVITY"
+        }"""),  cls.model)
+
+        cls.response_function.Initialize()
+        cls.response_function.CalculateValue()
+        cls.ref_value = cls.response_function.GetValue()
+        cls.response_function.CalculateGradient()
+
+
+        cls.element = cls.model_part.GetElement(1)
+
+    def testShapeSensitivity2D(self):
```

Comment:
- can you test the z-component of the sensitivity for the 2d element? (should be 0).
- please add a test for a 3D element as well.

Reply:
done :)